db_script.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO mysql.user (User,Host,authentication_string,ssl_cipher,x509_issuer,x509_subject) VALUES('demouser','localhost',PASSWORD('demopassword'),'','','');
```

Comment:
`PASSWORD()` is compatible only for MySQL 5 or less.  As the repository doesn't specific the version of MySQL (they released version 8) already.  This simply breaks the execution.